src/main/java/practice/StreamPractice.java

```diff
@@ -14,16 +19,27 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> Arrays.stream(s.split(",")).collect(Collectors.toList()))
+                .flatMap(s -> s.stream())
```

Comment:
```suggestion
                .flatMap(s -> Arrays.stream(s.split(","))
```

we dont need to create separate list here